roles/openshift_web_console/templates/console-smoke-test-pod.yaml.j2

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Pod
```

Comment:
Deployment so we can roll out new versions?

Reply:
We do have a `deployment` yaml in our repo, should be able to swap.